dataframe_to_postgresql.py

PinarKilic opened the pull request.

erkansirin78 commented:
Error-1: No module named 'sqlalchemy'
You need to add sqlalchemy to requirements.txt

Error-2: No module named 'psycopg2'
You need to add psycopg2-binary to requirements.txt

Error-3: It appends dataset repeatedly.
I run python dataframe_to_postgresql.py -t iris
and the result was:
traindb=> select count(1) from iris;
 count 
-------
 22500
(1 row)

PinarKilic commented:
added sqlalchemy to requirements.txt
added psycopg2-binary to requirements.txt
function transfers all data in one time because of using to_sql method has no feature to insert just one row

erkansirin78 commented:
It's improving. But, it still doesn't meet the desired functionality.
1. row sleep time doesn't work. 
2. What about PostgreSQL listens different port than 5432
3. Existing requirements kafka-python==2.0.2 pandas==1.0.5 pyarrow==1.0.0 don't exist.
